src/app/lib/layoutGrid.js

```diff
@@ -13,36 +13,43 @@ import {
 export const layoutGridWrapper = css`
   display: grid;
   @media (max-width: ${group3ScreenWidthMin}) {
-    grid-gap: ${GEL_SPACING}; /* Between grid items */
-    padding: 0 ${GEL_SPACING}; /* On grid wrapper */
+    grid-gap: 0 ${GEL_SPACING}; /* Between grid items */
   }
   @media (min-width: ${group3ScreenWidthMin}) {
-    grid-gap: ${GEL_SPACING_DBL}; /* Between grid items */
-    padding: 0 ${GEL_SPACING_DBL}; /* On grid wrapper */
+    grid-gap: 0 ${GEL_SPACING_DBL}; /* Between grid items */
   }
   @media (max-width: ${group3ScreenWidthMax}) {
-    grid-template-columns: repeat(6, 1fr);
+    grid-template-columns: repeat(6, [col] 1fr);
   }
   @media (min-width: ${group4ScreenWidthMin}) and (max-width: ${group4ScreenWidthMax}) {
-    grid-template-columns: repeat(8, 1fr);
+    /* 
+      Math: (1008 - (7*16))/8 = 112
+      Explainer: (width - 7 gaps of 16) / 8 columns = signle column width
+    */
+    grid-template-columns: [start] 1fr repeat(8, [col] minmax(0, 7rem)) 1fr [end];
   }
   @media (min-width: ${group5ScreenWidthMin}) {
-    grid-template-columns: repeat(10, 1fr);
+    /* 
+      Math: (1280 - (9*16))/10 = 113.6
+      Explainer: (width - 9 gaps of 16) / 10 columns = signle column width
+    */
+    grid-template-columns: [start] 1fr repeat(10, [col] minmax(0, 7.1rem)) 1fr [end];
```

Comment:
This creates the following https://codepen.io/phil-lee/pen/zmjaMO

![image](https://user-images.githubusercontent.com/7791726/47217129-45da5f00-d39f-11e8-83e1-8f2c732fe77a.png)

Reply:
Ahhh thanks for explaining that the lines are considered the Columns. I've updated the [Pen](https://codepen.io/phil-lee/full/zmjaMO/) to reflect this.

![image](https://user-images.githubusercontent.com/7791726/47499707-8da62e00-d858-11e8-8f16-eb05c33a205e.png)


Also agreed the naming should be reconsidered.